### ISSUE TITLE
Search modal background overlay not dark enough

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#177]
### Location
Search Modal close X

### Description
The X does not have sufficient contrast with its background. Please darken the background overly by 50% or so to ensure it conforms - I couldn't find what color it was to be able to specify another color since I'm assuming it's an rgba color.
